Jenkinsfile

```diff
@@ -60,7 +60,7 @@ node('mongodb-2.4') {
     }
 
     stage("Set up content schema dependency") {
-      govuk.contentSchemaDependency()
+      govuk.contentSchemaDependency(env.SCHEMA_BRANCH)
```

Comment:
Ok, I've updated this branch accordingly. I didn't remove the call, but the inclusion of SCHEMA_BRANCH in 9d521bb , can the entire call be removed?